apps/tx-processor/src/main.ts

```diff
@@ -52,9 +49,9 @@ async function run() {
   try {
     await Process();
   }
-  catch(e) {
+  catch(e: any) {
```

Comment:
Any 😂

Reply:
😛 it used to always be `any`, but someone changed it to `unknown` in the latest build of typescript

Reply:
😂 My favorite is still "noop"